### PR TITLE
Player should be able to choose not to mint

### PR DESCRIPTION
**Pull request by gopz**

### CODE REVIEW

**Comment by hphu** [comment]
Needs unit test for "None" selected by mint

File: sets/prosperity.py [hunk 1]
```diff
@@ -379,14 +379,12 @@ def play(self, skip=False):
...
-		else:
+		else:	
+			treasure_titles.append("None")
 			selection = yield self.played_by.select(1, 1, treasure_titles, "Choose a card to reveal")
-			self.game.announce("-- revealing " + self.game.log_string_from_title(selection[0]) + ", gaining a copy of it.")
-			yield self.played_by.gain(selection[0])
+			if selection[0] != 'None':
+				self.game.announce("-- revealing " + self.game.log_string_from_title(selection[0]) + ", gaining a copy of it.")
+				yield self.played_by.gain(selection[0])
 			crd.Card.on_finished(self, False, False)
```
Comment (hphu) [inline sets/prosperity.py]: you can do ```yield self.played_by.select(1, 1, treasure_titles, "Choose a card to reveal")``` instead of adding the "None" string manually, and checking if selection == [] instead

Comment (gopz) [inline sets/prosperity.py]: do you mean `yield self.played_by.select(None, 1, treasure_titles, "Choose a card to reveal")`
